tox.ini

```diff
@@ -1,22 +1,21 @@
 [tox]
 skip_missing_interpreters = True
 skipsdist = True
-usedevelop = True
 
 envlist =
-    py{27,33,34,35}-dj{18,19,110}-wt16
+    py{27,33,34,35}-dj{18,19,110}-wt{18,19,110,111}
```

Comment:
@OktayAltay, Now i can guess how the was passing with wagtail 1.8 that time. we defined `dj111 ` but not being used then all tests are using `wagtail 1.11` by default and tests are passing successfully.